fuzz/fuzz_all_native.py

```diff
@@ -19,6 +17,7 @@ def main():
     )
     parser.add_argument("--ci", required=False, help="In CI mode we run each parser only briefly once", action="store_true")
     parser.add_argument("--seconds", required=False, help="Run for this many seconds of total fuzz time before exiting")
+    parser.add_argument("--libroot", required=True, help="Path to the root of the library to fuzz")
```

Comment:
Does library mean package or module?

Reply:
That is a good question. I just used the same wording the script was already using which was LIB_ROOT. Something like go-project-root probably makes more sense.

Reply:
What does project mean? Package(s) (as in `./...`), module, and repo are all well known precise options.

Reply:
I guess repo doesn't make sense here because you could point it to a sub go project within a repo. So I guess module works. But basically what the script is looking for is where your go.mod file is. I have been calling them go projects incorrectly for a long time I guess.

Reply:
Updated: https://github.com/smartcontractkit/chainlink/pull/13827/commits/e106ee514fe726037a3c3a3e4cf35878efbc0a35